eslint.config.js

```diff
@@ -0,0 +1,28 @@
+const eslint = require('@eslint/js');
+const tseslint = require('@typescript-eslint/eslint-plugin');
+const tsparser = require('@typescript-eslint/parser');
+
+module.exports = [
+  {
+    files: ['**/*.ts', '**/*.tsx'],
+    languageOptions: {
+      parser: tsparser,
+      parserOptions: {
+        ecmaVersion: 'latest',
+        sourceType: 'module'
+      }
+    },
+    plugins: {
+      '@typescript-eslint': tseslint
+    },
+    rules: {
+      ...eslint.configs.recommended.rules,
+      ...tseslint.configs['eslint-recommended'].rules,
+      ...tseslint.configs.recommended.rules,
+      'no-extra-boolean-cast': 'off',
+      '@typescript-eslint/no-explicit-any': 'off',
+      '@typescript-eslint/no-unused-vars': 'warn',
+      'no-undef': 'warn'
```

Comment:
This is a bad practice but I'm putting this in just so we can get this out with updated dependencies.